test/new-e2e/tests/agent-subcommands/diagnose/diagnose_common_test.go

```diff
@@ -117,6 +117,11 @@ func (v *baseDiagnoseSuite) TestDiagnoseVerbose() {
 	assert.Contains(v.T(), diagnose, "connectivity-datadog-core-endpoints")
 }
 
+func (v *baseDiagnoseSuite) AssertOutputNotError(diagnose string) {
+	assert.NotContains(v.T(), diagnose, "FAIL")
```

Comment:
In the future, if we want to add other key word to exclude this format can be nice too ?
really not mandatory was just a thought.
for _, forbidden := range []string{"FAIL", "UNEXPECTED ERROR"} {
    assert.NotContains(v.T(), diagnose, forbidden)
}

Reply:
I think for two it is OK to not use a loop but for more it is definitely a good idea.